src/test/deploy/functions/tasks.spec.ts

```diff
@@ -0,0 +1,193 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+
+import * as tasks from "../../../deploy/functions/tasks";
+import { DeploymentTimer } from "../../../deploy/functions/deploymentTimer";
+import { ErrorHandler } from "../../../deploy/functions/errorHandler";
+import { FirebaseError } from "../../../error";
+
+describe("Function Deployment tasks", () => {
+  describe("functionsDeploymentHandler", () => {
+    const sandbox = sinon.createSandbox();
```

Comment:
It feels weird that this is initialized in global scope instead of beforeEach but reset in afterEach